draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -170,6 +161,7 @@ $negative-box-border-color: $color-red-300;
     color: $color-purple-800;
     outline: $color-blue-500 $border-focus-ring-border-style
       $border-focus-ring-border-width;
+    outline-offset: calc($border-focus-ring-border-width * -1);
```

Comment:
I can't seem to see this style in the browser... Is it working when you preview it from the Storybook link? @NatalieLoyzaga

Reply:
@lloydstubber I think because buildkite is failing the storybook preview hasn't updated...grr.

Reply:
Ahh true. Want to just throw a screenshot from your local dev here? Then I can give it a tick

Reply:
Yep done. Sorry should of done that anyways.